exotica_examples/CMakeLists.txt

```diff
@@ -51,13 +44,13 @@ install(DIRECTORY resources/ DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/res
 FILE(GLOB python_examples "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*")
 catkin_install_python(PROGRAMS ${python_examples} DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
+catkin_python_setup()
```

Comment:
I always thought this has to come _before_ `catkin_package()` but now I can't find this in the documentation anymore (cf. [here](http://docs.ros.org/melodic/api/catkin/html/user_guide/setup_dot_py.html))

Reply:
The docs say `When using the genmsg macro, ordering constraints exist`
and they provide this order:
```
project(...)
...
find_package(catkin ...)
...
catkin_python_setup()
...
generate_messages()
...
catkin_package()
```

The auto-generated CMakeLists also has it in that place.

Reply:
Thanks, I knew I read it somewhere but couldnt find the reference!